src/main/java/zbot/ui/MainWindow.java

```diff
@@ -0,0 +1,58 @@
+package zbot.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import zbot.ZBot;
```

Comment:
yes! I plan to do that but it will not pass the `checkstyle.xml` from AB3. I am still finding out how to edit the file. Thanks for pointing it out though!